base/stats.go

```diff
@@ -16,13 +52,39 @@ const (
 //to find valid property names
 //
 func init() {
-	StatsExpvars.Add("changesFeeds_total", 0)
-	StatsExpvars.Add("changesFeeds_active", 0)
-	StatsExpvars.Add("requests_total", 0)
-	StatsExpvars.Add("requests_active", 0)
-	StatsExpvars.Add("revisionCache_hits", 0)
-	StatsExpvars.Add("revisionCache_misses", 0)
+	for _, stat := range globalStatNames {
+		StatsExpvars.Add(stat, 0)
+	}
```

Comment:
I believe StatsExpvars collection was initializing these stats to zero specifically to handle telegraf/influx/grafana issues around non-existent expvars.  I don't think we should extend that to logging - it doesn't make sense to bloat the logs with zero-valued expvars.  (e.g. I don't care that changesFeeds_active is always zero because all my replications are blip).

Reply:
I agree with the reasoning, but I didn't think we could pull these out and break backwards compatibility.